requirements/lint.txt

```diff
@@ -1,8 +1,15 @@
 -r pip.txt
+# We need these special cases of Python < 3 for Read the Docs
+# corporate site.
 astroid==2.0.4; python_version > '3'
 astroid==1.6.4; python_version < '3'
-pylint==2.1.1
-pylint-django==2.0.2
+
+pylint==2.1.1; python_version > '3'
+pylint<2; python_version < '3'
+
+pylint-django==2.0.2; python_version > '3'
+pylint-django==0.11.1; python_version < '3'
```

Comment:
didn't know about that _flag_ D:

Reply:
I had no idea about it. Google was my friend :)